packages/imp.go

```diff
@@ -87,26 +87,20 @@ func (p *Importer) loadByExport(expfile string, pkgPath string) (pkg *types.Pack
 
 // ----------------------------------------------------------------------------
 
-type listExport struct {
-	Export string `json:"Export"`
-}
-
 // FindExport lookups export file (.a) of a package by its pkgPath.
 // It returns empty if pkgPath not found.
 func FindExport(dir, pkgPath string) (expfile string, err error) {
-	var ret listExport
 	data, err := golistExport(dir, pkgPath)
 	if err != nil {
 		return
 	}
-	err = json.Unmarshal(data, &ret)
-	expfile = ret.Export
+	expfile = strings.TrimSpace(string(data))
```

Comment:
Is `TrimSpace` needed?

Reply:
the data has `\n` end line

Reply:
use `expfile = string(bytes.TrimSuffix(data, []byte{'\n'}))`